test/core/integration/concrete/lockup-tranched/create-with-durations-lt/createWithDurationsLT.t.sol

```diff
@@ -51,14 +51,14 @@ contract CreateWithDurationsLT_Integration_Concrete_Test is Lockup_Tranched_Inte
     {
         uint40 startTime = getBlockTimestamp();
         LockupTranched.TrancheWithDuration[] memory tranches = defaults.tranchesWithDurations();
-        tranches[2].duration = 0;
-        uint256 index = 2;
+        uint256 index = 1;
+        tranches[index].duration = 0;
         vm.expectRevert(
             abi.encodeWithSelector(
                 Errors.SablierHelpers_TrancheTimestampsNotOrdered.selector,
                 index,
-                startTime + tranches[0].duration + tranches[1].duration,
-                startTime + tranches[0].duration + tranches[1].duration
+                startTime + tranches[0].duration,
```

Comment:
Rationale behind changing tranche size from 3 to 2?

Reply:
Previously, we declared the tranches with a size of 3 due to not having the "streaming" component in this distribution model, so that when warping to `WARP_26_PERCENT`, the withdrawable amount would be `WITHDRAW_AMOUNT`. Now, I have also updated the segments to have the first timestamp as `START_TIME + WARP_26_PERCENT` and can do the same for the tranches

previously segments:

https://github.com/sablier-labs/v2-core/blob/6a8ec1318f598c351606eb196b9ccebed7dedbab/test/utils/Defaults.sol#L143-L151

previously tranches:

https://github.com/sablier-labs/v2-core/blob/6a8ec1318f598c351606eb196b9ccebed7dedbab/test/utils/Defaults.sol#L176-L181

now they are in sync: 

https://github.com/sablier-labs/v2-core/blob/5764dd34aa56f544ff1e5caaa0be00bf9f9a610e/test/utils/Defaults.sol#L145-L157

https://github.com/sablier-labs/v2-core/blob/5764dd34aa56f544ff1e5caaa0be00bf9f9a610e/test/utils/Defaults.sol#L182-L186